assets/migrations/sqlite/005_initialize_schema.sql

```diff
@@ -0,0 +1,65 @@
+-- +goose Up
+CREATE TABLE tuple (
+    store CHAR(26) NOT NULL,
+    object_type VARCHAR(128) NOT NULL,
+    object_id VARCHAR(128) NOT NULL,
+    relation VARCHAR(50) NOT NULL,
+    user_object_type VARCHAR(128) NOT NULL,
+    user_object_id VARCHAR(128) NOT NULL,
+    user_relation VARCHAR(50) NOT NULL,
+    user_type VARCHAR(7) NOT NULL,
+    ulid CHAR(26) NOT NULL,
+    inserted_at TIMESTAMP NOT NULL,
+    condition_name VARCHAR(256),
+    condition_context LONGBLOB,
+    PRIMARY KEY (store, object_type, object_id, relation, user_object_type, user_object_id, user_relation)
+);
+
+CREATE UNIQUE INDEX idx_tuple_ulid ON tuple (ulid);
+CREATE INDEX idx_reverse_lookup_user ON tuple (store, object_type, relation, user_object_type, user_object_id, user_relation);
```

Comment:
If SQLLite supports partial indexes, could we bring these over from Postgres?

```go
CREATE INDEX idx_tuple_partial_user ON tuple (store, object_type, object_id, relation, _user) WHERE user_type = 'user';
CREATE INDEX idx_tuple_partial_userset ON tuple (store, object_type, object_id, relation, _user) WHERE user_type = 'userset';
```

These will make Check API more performant

Reply:
That should be possible, since SQLite does support partial indexes, I'll check out how that would work together with splitting `_user` up.